src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getUserSelector } from '../../store/selectors';
+import './CurrentUser.scss';
+import { clearSelectedUser } from '../../store/actions';
+
+export const CurrentUser: React.FC = () => {
+  const dispatch = useDispatch();
+  const user = useSelector((getUserSelector));
```

Comment:
i guess it's to much parentheses here
```suggestion
  const user = useSelector(getUserSelector);
```